packages/ebayui-core/src/components/ebay-filter-input/component.ts

```diff
@@ -0,0 +1,47 @@
+import type {
+    Input as TextboxInput,
+    TextboxEvent,
+} from "../ebay-textbox/component-browser";
+import type { WithNormalizedProps } from "../../global";
+import { validSizes } from "./constants";
+
+
+interface FilterInputInput extends Omit<TextboxInput, `on${string}`> {
```

Comment:
It seems we are not passing some non HTML attributes, like floating-label for example. Are we supposed to support those here also? If that is not the case we need to omit those from the input type also.